### ISSUE TITLE
Warning while `jekyll serve`

### ISSUE THREAD

**Issue by denpatin** [treffynnon/sqlstyle.guide#20]
Hi,

there's a weird build warning pursuing me any time I'm running the `jekyll serve` command.

```
Configuration file: /Users/denpatin/Repos/sqlstyle.guide/_config.yml
Configuration file: /Users/denpatin/Repos/sqlstyle.guide/_config.yml
            Source: /Users/denpatin/Repos/sqlstyle.guide
       Destination: /Users/denpatin/Repos/sqlstyle.guide/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
     Build Warning: Layout 'nil' requested in scripts.js does not exist.
     Build Warning: Layout 'nil' requested in style.css does not exist.
                    done in 1.119 seconds.
```

It doesn't affect the serving of the pages, however it still strings a bit up. Could you please check or explain, if you already know, what's going on here.

Thanks!

**Comment by treffynnon** [comment]
I wouldn't worry about that one. It's because those files don't have a layout specified as the css and js don't really need a Jekyll layout.

**Comment by treffynnon** [comment]
I've added a little pass through template to stop this warning from being shown - thanks.
